locallibrary/catalog/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+
+]
```

Comment:
chưa có define gì thêm thì để như sau em hè
```suggestion
urlpatterns = []
```